css/amp.css

```diff
@@ -862,6 +862,10 @@ amp-story .i-amphtml-loader {
   display: none !important;
 }
 
+amp-story-page:not(:first-of-type):not([distance]):not([active]) {
```

Comment:
Do we need `:not([distance]):not([active])` here?  If amp-story JS is not loaded, then these attributes won't be applied; if it is loaded, then this will be overridden anyway (right?)

Then shave a few bytes off of `amp.css`

Reply:
Good point, thanks. :)
I was using these selectors to make sure our implementation won't break because of CSS specificity. The previous rule would stop matching `amp-story-page`s as soon as the story code would run.

I went with a very simple CSS selector, and removed the `!important` statement, so we know our `amp-story.css` will override it no matter what.

PTAL